client/components/keyed-suggestions/index.jsx

```diff
@@ -108,6 +108,8 @@ class KeyedSuggestions extends React.Component {
 		} );
 	};
 
+	sanitizeInput = ( input ) => input.replace( /\W/g, '' );
```

Comment:
This is not i18n friendly. It will remove characters like `á` or `ñ`.

I'd suggest using https://github.com/sindresorhus/escape-string-regexp

Reply:
The thing with escaping here is that I'm not sure is the right action. If we escape, this will make the component work needlessly with a much longer string, while those special characters have no meaning in this context. That's why I preferred to remove them vs escaping them.

But you're right about the i18n concerns. I've updated the code to remove any special regex characters vs any non-latin-and-numbers ones.

Reply:
FWIW, there's the `deburr` function from lodash, which might have been useful here: https://lodash.com/docs/4.17.15#deburr (as a first step before rejecting special characters):

```
_.deburr('déjà vu');
// => 'deja vu'
```

Reply:
@klimeryk yeah, that would have been useful if we wanted to strip accented characters, but the thing is we want to keep them here, just in case any of the translations on that page contain such characters.

Reply:
Ah, good point! I misunderstood 🙇